javascript/modules/client/src/internal/interfaces/context.ts

```diff
@@ -38,7 +38,7 @@ interface Web3ContextState {
   network: Networkish;
   signer?: Signer;
   dao: string;
-  daoFactoryAddress: string;
+  daoFactoryAddress?: string;
```

Comment:
Shouldn't this be required on the state? In the constructor you will already know whether there is a daoFactoryAddress or not (default one)